packages/vscode/src/plugins/prisma-language-server/index.ts

```diff
@@ -77,19 +77,23 @@ const plugin: PrismaVSCodePlugin = {
       watcher = chokidar.watch(
         path.join(rootPath, '**/node_modules/.prisma/client/index.d.ts'),
         {
-          usePolling: false,
+          // ignore dotfiles (except .prisma) adjusted from chokidar README example
+          ignored: /(^|[\/\\])\.(?!prisma)./,
```

Comment:
I'm not clear on what `.prisma` is — is it a folder in `$HOME` we are using? Is it going to be in users' projects?

Reply:
See up for example

Default location of prisma-client-js is `node_modules/.prisma/client` folder and here we are looking for `node_modules/.prisma/client/index.d.ts` precisely.